ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -977,6 +994,40 @@ - (NSString *)stringForAnswer:(id)answer {
     return [self.impliedAnswerFormat stringForAnswer: @[answer]];
 }
 
++ (BOOL)supportsSecureCoding {
+    return YES;
+}
+
+- (instancetype)copyWithZone:(NSZone *)zone {
+    ORKBooleanAnswerFormat *answerFormat = [[[self class] allocWithZone:zone] initWithYesString:[_yes copy]
+                                                                                       noString:[_no copy]];
```

Comment:
This could break future changes if any properties are ever added to the base class. Instead, consider this:
````
- (instancetype)copyWithZone:(NSZone *)zone {
    ORKBooleanAnswerFormat *answerFormat = [super copyWithZone:zone];
    answerFormat->_yes = [_yes copy];
    answerFormat->_no = [_no copy];
    return answerFormat;
}
````